app/views/users/_form.html.erb

```diff
@@ -157,14 +156,60 @@
         <button class="btn btn-lg btn-primary btn-save" type="submit" tabindex="7"><%= t('users._form.save') %></button>
 
       <% end %>
-
+<script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.7/jquery.min.js"></script>
+<script type="text/javascript" src="js/noty/packaged/jquery.noty.packaged.min.js"></script>
       <script>
         (function() {
           $('.btn-save').click(function onClick(e) {
-            $(this).addClass("disabled") // disable the button after it is clicked
-                   .html("<i class='fa fa-spinner fa-spin'></i>"); // make a spinner that spins when clicked
+            
           })
         })();
+        /*
+       
+
+        */
+        function checkInputs() {
+          var input1 = document.getElementById("username").value;
+            var input2 = document.getElementById("email").value;
+            var input3 = document.getElementById("password").value;
+            var input4 = document.getElementById("password-confirmation").value;
+            var inputs = [input1, input2, input3, input4];
+            var fields = ["Username", "Email", "Password", "Password-confirmation"];
+            var inputsLength = inputs.length;
+            var emptyFields = [];
+            var i;
+            var errors=0;
+            /*
+space
+                */
+for(i=0;i<inputsLength;i++) {
+  if(inputs[i]==="") {
+  errors++;
+   emptyFields.push(fields[i]);
+  }
+  else {
+   // otherwise, no errors
+  }
+}
+//Now, we have detected all the empty fields and kept them in an array named emptyFields
+if(errors > 0) {
+    new Noty({
+                  type:'error',
+    text: "Please fill In: " + emptyFields,
+    animation: {
+        open: 'animated bounceInRight', // Animate.css class names
+        close: 'animated bounceOutRight',
+        timeout: 500 // Animate.css class names
+    }
+}).show();
+}
+else {
+   $(this).addClass("disabled") // disable the button after it is clicked
+                   .html("<i class='fa fa-spinner fa-spin'></i>"); // make a spinner that spins when clicked
+  console.log("Success ! No errors found...")
```

Comment:
Remove lines 190-192 and 209-210, they seem to be redundant. Also, please indent the code written by you according to the file. Thanks!

Reply:
190 - 192 is a piece of redundant code I've fixed that, but @gauravano  
In line 209: I am closing the function "checkInputs()"
And on line 210: I am closing the </script> tag which is necessary, because there is some html code under it,
Please tell me If I am wrong somewhere

Reply:
Probably, you are refering to updated code line numbers, but if you see in your prevoius commit, line 209 - `console.log` and line 210 - `alert` one. So, 211-212 acc to new code.

Reply:
@gauravano I've removed those lines, They were for testing when I was experimenting with the code earlier... Sorry for that